注意：return只能在function里面

//输出结果0,1,2
for(var i=0;i<5;i++){
    if(i == 3){
        break;
    }
    console.log(i,"break");
}
//输出0,1,2,4
for(var i=0;i<5;i++){
    if(i == 3){
        continue;
    }
    console.log(i,"continue");
}
//输出0,1,2
function returnDemo(){
    for(var i=0;i<5;i++){
        if(i == 3){
            return;
        }
        console.log(i,"return");
    }
}
returnDemo();


// 0 0 "break"
// 1 0 "break"
for(var i=0;i<2;i++){
    for(var j=0;j<3;j++){
        if(j == 1){
            break;
        }
        console.log(i,j,"break");
    }
}

// 0 0 "continue"
// 0 2 "continue"
// 0 3 "continue"
// 1 0 "continue"
// 1 2 "continue"
// 1 3 "continue"
for(var i=0;i<2;i++){
    for(var j=0;j<4;j++){
        if(j == 1){
            continue;
        }
        console.log(i,j,"continue");
    }
}

//0 0 "return"
function returnDemo2(){
    for(var i=0;i<2;i++){
        for(var j=0;j<4;j++){
            if(j == 1){
                return;
            }
            console.log(i,j,"return");
        }
    }
}
returnDemo2();

retrun true； 返回正确的处理结果。
return false；分会错误的处理结果，终止处理。
return；把控制权返回给页面。

示例：
function a（）{
   if（True）
       return false;
}
function Test（）{
   a（）;
   b（）;
   c（）;
}
即使a函数返回return false 阻止提交了，但是不影响 b（）以及 c（）函数的执行。在Test（）函数里调用a()函数，那面里面return false 对于Test（）函数来说，只是相当于返回值。而不能阻止Test（）函数执行。
总之：return false 只在当前函数有效，不会影响其他外部函数的执行